functions/artsy/index.js

```diff
@@ -276,6 +299,83 @@ app.intent('ShowsIntent', {
   }
 );
 
+app.intent('PodcastIntent', {
+    "slots": {
+      "NUMBER": "AMAZON.NUMBER"
+    },
+    "utterances": [
+      "{|to play} {|latest} podcast {|number|episode|episode number} {-|NUMBER}"
+    ]
+  },
+  function(req, res) {
+    var podcastNumber = req.slot('NUMBER');
+    var podcastStream = podcast.instance();
+    console.log(`app.PodcastIntent: ${podcastNumber || 'latest'}`);
+    if (podcastNumber && podcastNumber !== "") {
+      podcastStream = podcastStream.getEpisodeStreamById(podcastNumber);
+    } else {
+      podcastStream = podcastStream.getLatestEpisodeStream();
+    }
+
+    podcastStream.then(function(audioMpegEnclosure) {
+      var streamUrl = audioMpegEnclosure.url.replace('http://', 'https://'); // SSL required by Amazon, available on SoundCloud
+      var stream = {
+        url: streamUrl,
+        token: streamUrl,
+        offsetInMilliseconds: 0
+      }
+      res.audioPlayerPlayStream('REPLACE_ALL', stream);
```

Comment:
Yes, this is a "native" function of the API of alexa-app, see https://github.com/alexa-js/alexa-app